docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,181 @@
+# ServiceWaterHeating - Rule 11-10
+
+**Schema Version:** 0.0.37  
+**Mandatory Rule:** True  
+**Rule ID:** 11-10  
+
+**Rule Description:** The service water heating system type in the baseline building design shall match the minimum efficiency requirements in Section 7.4.2  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b  
+
+**Evaluation Context:** Each ServiceWaterHeatingEquipment  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_SHW_equipment_associated_with_each_SWH_bat()
+
+**Applicability Checks:**
+- Each SHW Equipment is applicable
+- use get_SHW_equipment_associated_with_each_SWH_bat to get the SHW BATs and SHW use in the building: `shw_bats_and_equip_dict = get_SWH_components_associated_with_each_SWH_bat(B_RMD)`
+- look at each SHW equipment in the building: `for swh_eq in B_RMD.service_water_heating_equipment:`
+  - CONTINUE TO RULE LOGIC
+- there are no cases where a Service Water Heating Equipment is not applicable to the rule
+- if there are no SWH equipment in the RMD, then this rule will not be evaluated
+
+
+    ## Rule Logic:
+  - create a value to track the rule status: `rule_status = "FAIL"`
+  - create a variable to track any notes: `rule_note = """`
+  - find the swh_bat by looking at each swh_bat in shw_bats_and_equip_dict, and looking for the swh_eq.id in the list of SWHHeatingEq for each BAT.  Start by setting the swh_bat to "NONE: `swh_bat = "NONE"`
+  - look at each swh building area type: `for swh_bat1 in shw_bats_and_equip_dict:`
+    - look for the id of the swh_eq in the SWHHeatingEq list: `if swh_eq.id in? shw_bats_and_equip_dict[swh_bat1]["SWHHeatingEq"]:`
+      - set the swh_bat to swh_bat1: `swh_bat = swh_bat1`
+      - and break out of the loop: `break`
+  - check if the swh_bat is still NONE we cannot evaluate the rule, create a note, and go to rule assertion: `if swh_bat == "NONE"; rule_note = "We were not able to identify the building area type for this service water heater, so the rule will not be evaluated."; GO TO RULE ASSERTION`
```

Comment:
WHy do we need the BAT to evaluate this rule? I am not understandign that. Seems like we can just loop through all the SWH equipment and leave BAT out of it.

Reply:
I think the intent was also to check that the system type was correct based on Table G3.1.1-2, but rule 11-7 includes this check, so I've removed the logic about BAT.